pkg/nsx/services/securitypolicy/expand.go

```diff
@@ -147,9 +147,7 @@ func (service *SecurityPolicyService) resolveNamedPort(obj *v1alpha1.SecurityPol
 	}
 
 	if len(portAddress) == 0 {
-		return nil, nsxutil.NoFilteredPod{
-			Desc: "no pod has the corresponding named port, please check the pod selector or labels of CR",
-		}
+		log.Info("warning, no pod has the corresponding named port")
```

Comment:
if there is no pods has the named port discovered. There should be two cases:
1. creating named port security policy,  podSelector don't find corresponding pod with named port.
2. deleting the pods with named port, so the previous ipSet group should clear existing ip members. 

do we need to make different handing for both cases? currently, if portAddress == 0,  and then,
the groups in the rule should have empty members ,is it?

Reply:
updated.